doc/quick-start/qs-guide.rst

```diff
@@ -232,7 +232,149 @@ If all the steps were followed correctly, it will be possible to see that the KN
 
 ----------------------------------------------------------------
 
-KNoT Cloud SDK
---------------
+Creating an Application on KNoT Cloud
+-------------------------------------
 
-To interact with data from device, utilize KNoT Cloud SDK to construct a User Application.
+#. Log-in on KNoT Cloud UI.
+
+.. figure:: ../../_static/cloud_login.png
+   :scale: 35 %
+   :alt: KNoT Cloud UI login page
+   :align: center
+
+#. Go to Apps page and create an application.
+
+.. figure:: ../../_static/cloud_app_creation.png
+   :scale: 25 %
+   :alt: KNoT Cloud UI applications page
+   :align: center
+
+#. Set a name to your application.
+
+.. figure:: ../../_static/cloud_app_creation_modal.png
+   :scale: 28 %
+   :alt: KNoT Cloud UI applications modal
+   :align: center
+
+#. Download the application credentials.
+
+.. figure:: ../../_static/cloud_app_credentials.png
+   :scale: 35 %
+   :alt: KNoT Cloud UI download app credentials
+   :align: center
+
+#. The application credentials should looks like:
+
+    .. code-block:: json
+
+      {
+        "type": "knot:app",
+        "metadata": {
+          "name": "Hello Application"
+        },
+        "knot": {
+          "id": "3c92790f-f265-46c9-bbf8-e440f0447587",
+          "isThingManager": false
+        },
+        "token": "826faa7d545e39c8b2a198c74d0da54f95dfea55"
+      }
+
+----------------------------------------------------------------
+
+Interacting with Application through KNoT Cloud SDK
+---------------------------------------------------
+
+#. Create a new directory and start a NodeJS application on it.
+
+    .. code-block:: bash
+
+      npm init -y
+
+#. Install the KNoT Cloud SDK for JavaScript.
+
+    .. code-block:: bash
+
+      npm i -s @cesarbr/knot-cloud-sdk-js
+
+#. Create a ``index.js`` file and import the ``knot-cloud-sdk-js``.
+
+   .. code-block:: javascript
+
+      const { Client } = require('@cesarbr/knot-cloud-sdk-js');
+
+#. Create a client connection instance with the KNoT Cloud WebSocket server by using the application credentials that you have obtained above.
+
+   .. code-block:: javascript
+
+    const client = new Client({
+      hostname: 'ws.knot.cloud',
+      protocol: 'wss',
+      port: 443,
+      pathname: '/ws',
+      id: '3c92790f-f265-46c9-bbf8-e440f0447587', // APP ID
+      token: '826faa7d545e39c8b2a198c74d0da54f95dfea55', // APP TOKEN
+    });
+
+#. Send ``setData`` command to turn off the KNoT Dongle's LED (you can get it's ID from the gateway interface).
+
+   .. code-block:: javascript
+
+      const data = [
+        {
+          sensorId: 0, // LED's sensorID
+          value: false, // New LED's value
+        },
+      ];
+
+      client.on('ready', () => {
+        client.setData('2828b4c983f2d9d1', data); // Send setData command, passing to it the KNoT Dongle's ID and the data.
```

Comment:
Please, use **Thing** instead of **Dongle**

Reply:
Done.